portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTests.macro

```diff
@@ -0,0 +1,31 @@
+definition {
+
+	macro deleteABTest {
+		var key_testName = "${testName}";
+
+		Click(locator1 = "ACTests#AB_TEST_NAME");
+
+		Click(locator1 = "ACTests#AB_ELLIPSIS");
+
+		Click(locator1 = "ACTests#DELETE_TEST");
+
+		Button.clickDelete();
+	}
+
+	macro statusABTest {
+		AssertTextEquals(
+			locator1 = "ACTests#LABEL_STATUS",
+			value1 = "${status}");
+	}
+
+	macro terminateABTest {
+		var key_testName = "${testName}";
+
+		Click(locator1 = "ACTests#AB_TEST_NAME");
```

Comment:
Hey @joshitagaki, I think you could use:

`ACNavigation.openItem(itemName = "${testName}");`

What do you think?